### ISSUE TITLE
REPL prompt display error when executing code from the text editor (Shift Enter)

### ISSUE THREAD

**Issue by chenspc** [julia-vscode/julia-vscode#3303]
Hi, I noticed that when using Julia 1.9 in VS Code, if I execute a line of code from the text editor (using Shift Enter), the `julia>` prompt would change to `#87`. This is not just a purely cosmetic problem, because it also causes multiline statements to hang. 

```
julia> using HDF5 # from REPL

#87using HDF5 # from file

#87a = Complex(1,                       # Hanging... See the 1.8 example for the expected output

```

Julia 1.8

```
julia> using HDF5 # from REPL

julia> using HDF5 # from file

julia> a = Complex(1, 
           2)
1 + 2im

julia> 
```

**Comment by harrysw1729** [comment]
We somehow posted the same issue exactly at the same time. :) #3302

**Comment by chenspc** [comment]
lol, I also just realised this :)

**Comment by chenspc** [comment]
@harrysw1729 thanks for the workaround. It works!

**Comment by chenspc** [comment]
Resolved after 1.47.2
